.golangci.yml

```diff
@@ -0,0 +1,79 @@
+linters-settings:
```

Comment:
@Dmdv I think this file should be in different branch

Reply:
Fixed